### ISSUE TITLE
contains virus sprisky.U! dowloaded from electrum-ltc.org/

### ISSUE THREAD

**Issue by ONYXARC** [pooler/electrum-ltc#175]
Says contains virus sprisky.U! dowloaded from electrum-ltc.org/... so if i can't trust it from there then i don't need one or do i need litecoin. this explains some of the other issues being posted here like missing coins and such.  MY question is whether this virus was placed in it from here or from the litecoin people.

**Comment by pooler** [comment]
If you downloaded the file from the official site and the signature is correct, then this is almost certainly a false positive, as is often the case with cryptocurrency software. This is nothing specific to Electrum-LTC, either: Electrum has the same problem and is recognized as a threat by multiple antivirus programs (for instance, [this](https://www.virustotal.com/#/file/d42391fe8069be5ebfab9d3e1e4f4f8dbd0e36d80bc841677d76359510e22b6e/detection) is the latest version of Electrum for Bitcoin; 8 engines detect it as malware). Several tickets have been opened about this in the upstream issue tracker, but apart from reporting the false positives to antivirus vendors, there doesn't seem to be much that can be done.

**Comment by ONYXARC** [comment]
I had downloaded several Electrum clients from other coins with no virus detection, scans daily with no updates being flagged.. doesnt make much sense

**Comment by pooler** [comment]
I cannot speak for other forks of Electrum. The digital signatures say that the binaries of Electrum-LTC on the website have not been tampered with. Moreover, a VirusTotal scan of the Windows binaries of Electrum-LTC 3.1.2.1 gives just the usual few false positives (and even fewer than Electrum 3.1.2 as of this writing).

If you do not trust the binaries I made, let me remind you that Electrum-LTC is open source. You can always download, verify, and run the source code. And even make your own binaries.

**Comment by ONYXARC** [comment]
It is just concerning to have downloaded electrum wallet and get virus alert after having downloaded numerous others with no suspect code. Coupled with weeks of hearing complaints of missing LTC from people and countless posts it makes me very cautious and wary as these missing coins are directly from wallets. I greatly appreciate your work. You are a magnificent coder and your contributions are priceless in my opinion. I apologize for being a bit on edge. With the explosion in scams and malicious code recently I just see so many more victims and even last week had an address that I copied paste as a different address multiple times before realize I had picked up something somewhere undetected. Good luck in your endeavors and thank you for your programs.
